source/common/buffer/buffer_impl.h

```diff
@@ -153,7 +154,9 @@ class Slice : public SliceData {
     uint8_t* dest = base_ + reservable_;
     reservable_ += copy_size;
     // NOLINTNEXTLINE(clang-analyzer-core.NullDereference)
-    memcpy(dest, data, copy_size);
+    MemBlockBuilder<uint8_t> mem_builder(dest, copy_size);
+    uint8_t* data_non_raw = reinterpret_cast<uint8_t*>(const_cast<void*>(data));
```

Comment:
you shouldn't need to const_cast here. You can just reinterpret_cast to `const uint8_t*`

Reply:
Sure, I tryied to use absl::Span<uint8_t> constructor, that is why I removed the const. I will see to fix this.